tests/test_fluentd.py

```diff
@@ -0,0 +1,61 @@
+from tests.helm_template_generator import render_chart
+import pytest
+from . import supported_k8s_versions
+import jmespath
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+def test_fluentd_daemonset(kube_version):
+    """Test that helm renders a good DaemonSet template for fluentd when rbacEnabled=True."""
```

Comment:
Could this be changed to "Test that helm renders a volume mount for private ca certificates for fluentd daemonset when private-ca-certificates are enabled"?